saleor/product/models/base.py

```diff
@@ -235,6 +241,7 @@ def get_first_category(self):
 
     def is_available(self):
         # type: () -> bool
+        """Checks if product is available today for customers"""
```

Comment:
I'd also consider moving this to the availability info object but it's not a documentation issue.